app/views/users/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> Users index </h1>
```

Comment:
I think you should not expose this view, users should not be able to see a list of all the other users ;)